src/main/java/dataMgmt/SessionStorage.java

```diff
@@ -1,21 +1,31 @@
 package dataMgmt;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Observable;
+import com.google.common.collect.BiMap;
+import websocket.models.Project;
+
+import java.util.*;
 
 /**
  * The storage unit for session data.
  * Created by fahslaj on 5/3/2016.
  */
 public class SessionStorage extends Observable {
```

Comment:
yes. it needs to be observable because if things change in session storage, the UI may need to change. Tomorrow [Sunday the 16th] at our meeting, Gene and I will explain why this necessary in detail.